cmd/grafana-agent-service/config_windows.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"golang.org/x/sys/windows/registry"
+)
+
+// config holds configuration options to run the service.
+type config struct {
+	// ServicePath points to the path of the managed Grafana Agent binary.
+	ServicePath string
+
+	// Args holds arguments to pass to the Grafana Agent binary. os.Args[0] is
+	// not included.
+	Args []string
+
+	// WorkingDirectory points to the working directory to run the Grafana Agent
+	// binary from.
+	WorkingDirectory string
+}
+
+// loadConfig loads the config from the Windows registry.
+func loadConfig() (*config, error) {
+	agentKey, err := registry.OpenKey(registry.LOCAL_MACHINE, `Software\Grafana\Grafana Agent`, registry.READ)
```

Comment:
Should this be `Software\Grafana\Grafana Agent Flow`? 

cc @erikbaranowski @mattdurham

Reply:
I'd say yes. It will be more obvious that this custom installer created stuff when debugging plus it doesn't have the flow/static installer bumping into each other.

If you do, reminder to update `install_script.nsis`

Reply:
I'll change to `Grafana Agent Flow`, we can always change it again later and have some kind of migration or support both paths.